manim/mobject/types/image_mobject.py

```diff
@@ -63,15 +63,16 @@ def set_resampling_algorithm(self, resampling_algorithm):
 
         Parameters
         ----------
-        resampling_algorithm : :class:`int`, an integer constant described in the
-        Pillow library, or one from the RESAMPLING_ALGORITHMS global dictionary, under
-        the following keys:
-         * 'bicubic' or 'cubic'
-         * 'nearest' or 'none'
-         * 'box'
-         * 'bilinear' or 'linear'
-         * 'hamming'
-         * 'lanczos' or 'antialias'
+        resampling_algorithm
+            An integer constant described in the Pillow library,
+            or one from the RESAMPLING_ALGORITHMS global dictionary,
+            under the following keys:
+            * 'bicubic' or 'cubic'
```

Comment:
See https://manimce--2988.org.readthedocs.build/en/2988/reference/manim.mobject.types.image_mobject.AbstractImageMobject.html#manim.mobject.types.image_mobject.AbstractImageMobject.set_resampling_algorithm -- if this should be formatted as a list, there has to be a newline between lines 69 and 70.